### ISSUE TITLE
My Alexa Routes are not found (Laravel)

### ISSUE THREAD

**Issue by luebbert42** [develpr/alexa-app#44]
I am trying to get a "hello world" up and running

- Laravel 5.5
- i have configured aliases and providers as documented
- the alexa.php config file is in my config.folder
- regular routes work

in web.php I have included:
\AlexaRoute::launch('/your-app-uri', 'App\Http\Controllers\AnyController@anyMethod');

And created AnyController and anyMethod.

When I launch /your-app-uri I am getting a 404. Same when I post something against this URL using Postman. artisan route:list does not show your-app-uri as valid url.

Is this supposed to be like this? Did I miss anything to get the AlexaRouter working?

**Comment by S43534** [comment]
Maybe https://github.com/develpr/alexa-app/pull/41/commits/9423ec3e79ddcc239eebe6311110f90b3e48d9b4?

Sebastian

**Comment by vrajroham** [comment]
@luebbert42 Try registering following middleware and see the stacktrace in logs. Let us know what is in stacktrace. Here you can verify your each request and it's parameters.
```php
<?php

namespace App\Http\Middleware;

use Closure;
use Illuminate\Support\Facades\Log;

class LogAfterRequest
{
    public function handle($request, Closure $next)
    {
        return $next($request);
    }

    public function terminate($request, $response)
    {
        Log::info('app.request', ['request' => $request->all()]);
        // Log::info('app.response', ['response' => json_encode($response->content())]);
    }

    private function getallheaders()
    {
        if (!is_array($_SERVER)) {
            return [];
        }

        $headers = [];
        foreach ($_SERVER as $name => $value) {
            if (substr($name, 0, 5) == 'HTTP_') {
                $headers[str_replace(' ', '-', ucwords(strtolower(str_replace('_', ' ', substr($name, 5)))))] = $value;
            }
        }

        return $headers;
    }
}

```

**Comment by luebbert42** [comment]
issue = bogus

Explanation:
The lib seems to return 404 if the request is not valid (I kind of expected another status code in this case). 

I wanted to do a quick smoke test if I configured everything correctly and sent a simple "foo=bar" post request via Postman. 

To do my smoke test I captured the alexa request using requestb.in and posted the JSON as raw request to my url using Postman. This worked. 

Thanks for your support, guys, anyway :-)
